### ISSUE TITLE
pypi

### ISSUE THREAD

**Issue by mnot** [ietf-tools/idnits#9]
Pypi [lists](https://pypi.org/project/idnits/) idnits at version 3, but this repos seems to have 2.17.1.

**Comment by rjsparks** [comment]
That is an abandoned attempt at re-implementing idnits, and I'll go clean it away. We _do_ expect a new idnits in the next month or so, at which point this will all be OBE.

**Comment by Flowdalic** [comment]
Is https://github.com/ietf-tools/rfclint the designated idnits successor?

**Comment by HenkVanMaanen** [comment]
> Is https://github.com/ietf-tools/rfclint the designated idnits successor?

Would like to know this too

**Comment by rjsparks** [comment]
No, it is not - they are separate tools. The idnits successor, when it exists, will be called idnits3. (There is an older start at idnits3 in pypi that will be overwritten, hopefully before the year ends).
